pkg/runtime/application_config_provider.go

```diff
@@ -22,6 +23,7 @@ const MB = KB * KB
 var flyteAdminConfig = config.MustRegisterSection(flyteAdmin, &interfaces.ApplicationConfig{
 	ProfilerPort:          metricPort,
 	MetricsScope:          "flyte:",
+	MetricKeys:            []string{contextutils.ProjectKey.String(), contextutils.DomainKey.String()},
```

Comment:
I think the default should have a few more, right? WorkflowID, taskID, phase... etc. should probably continue to be there... (to keep close to current behavior)... deployments that don't need this granularity can choose to override the config...

Reply:
ok, done